lib/node_modules/@stdlib/blas/base/sspr/README.md

```diff
@@ -183,21 +183,60 @@ console.log( AP );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/sspr.h"
+#include "stdlib/blas/base/shared.h"
 ```
 
-#### TODO
+#### c_sspr( order, uplo, N, alpha, \*X, strideX, \*AP )
 
-TODO.
+Performs the symmetric rank 1 operation `A = α*x*x^T + A` where `α` is a scalar, `x` is an `N` element vector, and `A` is an `N` by `N` symmetric matrix supplied in packed form.
+
+```c
+float AP[] = { 1.0f, 2.0f, 3.0f, 1.0f, 2.0f, 1.0f };
+const float x[] = { 1.0f, 2.0f, 3.0f };
+
+c_sspr( CblasColMajor, CblasUpper, 3, 1.0f, x, 1, AP );
+```
+
+The function accepts the following arguments:
+
+-   **order**: `[in] CBLAS_LAYOUT` storage layout.
+-   **uplo**: `[in] CBLAS_UPLO` specifies whether the upper or lower triangular part of the symmetric matrix `A` should be referenced.
+-   **N**: `[in] CBLAS_INT` number of elements along each dimension of `A`.
+-   **alpha**: `[in] float` scalar.
+-   **X**: `[in] float*` first input array.
+-   **strideX**: `[in] CBLAS_INT` index increment for `X`.
+-   **AP**: `[inout] float*` packed form of a symmetric matrix `A`.
 
 ```c
-TODO
+void c_sspr( const CBLAS_LAYOUT order, const CBLAS_UPLO uplo, const CBLAS_INT N, const float alpha, const float *X, const CBLAS_INT strideX, float *AP )
 ```
 
-TODO
+#### c_sspr_ndarray( order, uplo, N, alpha, \*X, strideX, \*AP, strideAP, offsetAP )
+
+Performs the symmetric rank 1 operation `A = α*x*x^T + A` where `α` is a scalar, `x` is an `N` element vector, and `A` is an `N` by `N` symmetric matrix supplied in packed form using alternative indexing semantics.
+
+```c
+float AP[] = { 1.0f, 2.0f, 3.0f, 1.0f, 2.0f, 1.0f };
+const float x[] = { 1.0f, 2.0f, 3.0f };
+
+c_sspr_ndarray( CblasColMajor, CblasUpper, 3, 1.0f, x, 1, AP, 1, 0 );
+```
+
+The function accepts the following arguments:
+
+-   **order**: `[in] CBLAS_LAYOUT` storage layout.
+-   **uplo**: `[in] CBLAS_UPLO` specifies whether the upper or lower triangular part of the symmetric matrix `A` should be referenced.
+-   **N**: `[in] CBLAS_INT` number of elements along each dimension of `A`.
+-   **alpha**: `[in] float` scalar.
+-   **X**: `[in] float*` first input array.
+-   **strideX**: `[in] CBLAS_INT` index increment for `X`.
```

Comment:
We should be consistent in our descriptions (e.g., w L235). "stride length for `X`"